doc/finalResult.js

```diff
@@ -22,11 +22,11 @@ function parseUl(data, device) {
                 value: pair[1],
                 type: findType(pair[0])
             };
-        
+
         return attribute;
     }
-    
-    return data.split("#").map(createAttribute);
+
+    return data.split('#').map(createAttribute);
```

Comment:
I understand that single quote convention is the one we mainly use along .js code except for the few cases fixed in this PR.

Is my understanding correct?

Reply:
NTC